bindings/node.js/lib/kzg.js

```diff
@@ -43,6 +45,7 @@ bindings.loadTrustedSetup = function loadTrustedSetup(filePath) {
   if (path.parse(filePath).ext === ".json") {
     filePath = transformTrustedSetupJson(filePath);
   }
+  console.log(filePath);
```

Comment:
This will show in the client during startup.  It may be better to denote what it represents if you actually want this logged so it's clear what it is in the system logs.
```suggestion
  console.log(`c-kzg trusted setup filepath: ${filePath}`);
```

Reply:
Yeah, removed that log. I was debugging a problem when loading the trusted setup.

Reply:
🥷